app/src/main/res/values/strings.xml

```diff
@@ -5002,5 +5002,11 @@
   <string name="AccountDeleted">Account deleted</string>
   <string name="AccountDeletedText">Your account %1$s (%2$s) is now deleted from Telegram servers, and can no longer be restored.\n\nUsing the same phone number will create a new account.</string>
   <string name="ViewOnce">View once</string>
+  <string name="ConvertToBroadcastGroup">Convert to Broadcast Group</string>
+  <string name="ConvertToBroadcastGroupDesc">Broadcast groups can have over 200,000 members, but only admins can send messages in them.</string>
```

Comment:
Plural with `tdlib.supergroupMaxSize()` must be used instead of constant `200,000` here.